routes/story.js

```diff
@@ -1,39 +1,61 @@
 const express = require("express");
 const router = express.Router();
+const Story = require('../models/Story');
 
 // @desc Story summary
 // @route GET /story/:storyId
 // @access Public
 router.get("/:storyId", (req, res) => {
-  res.send("fetch story details with that params.storyId");
+    const { storyId } = req.params.storyId;
+    try {
+        Story.findOne({ _id: storyId }, (err, story) => {
+            if (!story) throw "Invalid Story ID";
+        }).lean();
+        res.send("fetch story details with that params.storyId");
+    } catch (err) {
+        console.log(err);
```

Comment:
In case of error, you should send error 500 with some response message like "Internal server error".

Reply:
Changes incorporated